packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1293,12 +1293,16 @@ mixin WidgetInspectorService {
       callback: _getSelectedWidget,
       registerExtension: registerExtension,
     );
+    _registerServiceExtensionWithArg(
+      name: WidgetInspectorServiceExtensions.getSelectedLocalWidget.name,
+      callback: _getSelectedLocalWidget,
+      registerExtension: registerExtension,
+    );
     _registerServiceExtensionWithArg(
       name: WidgetInspectorServiceExtensions.getSelectedSummaryWidget.name,
```

Comment:
how does this new api differ from this one?

Reply:
Hahaha I was very tunnel visioned and did not notice this. Looking at the code, I'm guessing they do the same thing. Let me give that one a try and if so I will close this PR 🙃

Reply:
`getSelectedSummaryWidget` works exactly the same as this new API, so I will close this PR 🤦‍♀️